### ISSUE TITLE
Currently SMS/Voice quota limit only allowed not email in Notification Policy

### ISSUE THREAD

**Issue by kasirajans** [pingidentity/terraform-provider-pingone#716]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

Thank you for opening an issue. Please note that we try to keep the Terraform issue tracker reserved for bug reports and feature requests. For general usage questions, please see: https://www.terraform.io/community.html.

### PingOne Terraform provider Version
pingidentity/pingone - 0.24

### Terraform Version
Terraform v1.6

### Affected Resource(s)
- pingone_notification_policy 

### Terraform Configuration Files
```hcl
resource "pingone_notification_policy" "Default_Notification_Policy" {
  country_limit = {
    countries        = ["AF"]
    delivery_methods = ["SMS", "Voice"]
    type             = "DENIED"
  }
  environment_id = var.pingone_environment_id
  name           = "Default Notification Policy"
  quota {
    total  =<count>
    type   = "USER"
    unused = null
    used   = null
  }
  quota {
    total  =<count>
    type   = "USER"
    unused = null
    used   = null
  }
}
```

### Debug Output
2024-01-12T16:55:55.032-0700 [INFO]  Terraform version: 1.6.6
2024-01-12T16:55:55.033-0700 [DEBUG] using github.com/hashicorp/go-tfe v1.36.0
2024-01-12T16:55:55.033-0700 [DEBUG] using github.com/hashicorp/hcl/v2 v2.19.1
2024-01-12T16:55:55.033-0700 [DEBUG] using github.com/hashicorp/terraform-svchost v0.1.1
2024-01-12T16:55:55.033-0700 [DEBUG] using github.com/zclconf/go-cty v1.14.1
2024-01-12T16:55:55.033-0700 [INFO]  Go runtime version: go1.21.5
2024-01-12T16:55:55.033-0700 [INFO]  CLI args: []string{"/Users/<id>/binaries/terraform/terraform", "plan", "-target=pingone_notification_policy.Default_Notification_Policy"}
2024-01-12T16:55:55.033-0700 [DEBUG] Attempting to open CLI config file: /Users/<id>/.terraformrc
2024-01-12T16:55:55.033-0700 [DEBUG] File doesn't exist, but doesn't need to. Ignoring.
2024-01-12T16:55:55.033-0700 [DEBUG] ignoring non-existing provider search directory terraform.d/plugins
2024-01-12T16:55:55.033-0700 [DEBUG] ignoring non-existing provider search directory /Users/<id>/.terraform.d/plugins
2024-01-12T16:55:55.033-0700 [DEBUG] ignoring non-existing provider search directory /Users/<id>/Library/Application Support/io.terraform/plugins
2024-01-12T16:55:55.033-0700 [DEBUG] ignoring non-existing provider search directory /Library/Application Support/io.terraform/plugins
2024-01-12T16:55:55.033-0700 [INFO]  CLI command args: []string{"plan", "-target=pingone_notification_policy.Default_Notification_Policy"}
2024-01-12T16:55:55.036-0700 [DEBUG] New state was assigned lineage "8baee620-e868-2e91-cab2-26171a5913d0"
2024-01-12T16:55:55.280-0700 [DEBUG] checking for provisioner in "."
2024-01-12T16:55:55.280-0700 [DEBUG] checking for provisioner in "/Users/<id>/binaries/terraform"
2024-01-12T16:55:55.282-0700 [INFO]  backend/local: starting Plan operation
2024-01-12T16:55:55.286-0700 [DEBUG] created provider logger: level=debug
2024-01-12T16:55:55.286-0700 [INFO]  provider: configuring client automatic mTLS
2024-01-12T16:55:55.292-0700 [DEBUG] provider: starting plugin: path=.terraform/providers/registry.terraform.io/hashicorp/random/3.6.0/darwin_arm64/terraform-provider-random_v3.6.0_x5 args=[".terraform/providers/registry.terraform.io/hashicorp/random/3.6.0/darwin_arm64/terraform-provider-random_v3.6.0_x5"]
2024-01-12T16:55:55.294-0700 [DEBUG] provider: plugin started: path=.terraform/providers/registry.terraform.io/hashicorp/random/3.6.0/darwin_arm64/terraform-provider-random_v3.6.0_x5 pid=56388
2024-01-12T16:55:55.294-0700 [DEBUG] provider: waiting for RPC address: path=.terraform/providers/registry.terraform.io/hashicorp/random/3.6.0/darwin_arm64/terraform-provider-random_v3.6.0_x5
2024-01-12T16:55:55.360-0700 [INFO]  provider.terraform-provider-random_v3.6.0_x5: configuring server automatic mTLS: timestamp=2024-01-12T16:55:55.360-0700
2024-01-12T16:55:55.366-0700 [DEBUG] provider: using plugin: version=5
2024-01-12T16:55:55.366-0700 [DEBUG] provider.terraform-provider-random_v3.6.0_x5: plugin address: address=/var/folders/lk/rdfj6whd5cd2vk0pkhv6zf9wfb1nt4/T/plugin3004932502 network=unix timestamp=2024-01-12T16:55:55.366-0700
2024-01-12T16:55:55.372-0700 [DEBUG] No provider meta schema returned
2024-01-12T16:55:55.373-0700 [DEBUG] provider.stdio: received EOF, stopping recv loop: err="rpc error: code = Unavailable desc = error reading from server: EOF"
2024-01-12T16:55:55.373-0700 [DEBUG] provider: plugin process exited: path=.terraform/providers/registry.terraform.io/hashicorp/random/3.6.0/darwin_arm64/terraform-provider-random_v3.6.0_x5 pid=56388
2024-01-12T16:55:55.373-0700 [DEBUG] provider: plugin exited
2024-01-12T16:55:55.373-0700 [DEBUG] created provider logger: level=debug
2024-01-12T16:55:55.374-0700 [INFO]  provider: configuring client automatic mTLS
2024-01-12T16:55:55.376-0700 [DEBUG] provider: starting plugin: path=.terraform/providers/registry.terraform.io/pingidentity/pingone/0.25.0/darwin_arm64/terraform-provider-pingone_v0.25.0 args=[".terraform/providers/registry.terraform.io/pingidentity/pingone/0.25.0/darwin_arm64/terraform-provider-pingone_v0.25.0"]
2024-01-12T16:55:55.378-0700 [DEBUG] provider: plugin started: path=.terraform/providers/registry.terraform.io/pingidentity/pingone/0.25.0/darwin_arm64/terraform-provider-pingone_v0.25.0 pid=56389
2024-01-12T16:55:55.378-0700 [DEBUG] provider: waiting for RPC address: path=.terraform/providers/registry.terraform.io/pingidentity/pingone/0.25.0/darwin_arm64/terraform-provider-pingone_v0.25.0
2024-01-12T16:55:55.477-0700 [INFO]  provider.terraform-provider-pingone_v0.25.0: configuring server automatic mTLS: timestamp=2024-01-12T16:55:55.476-0700
2024-01-12T16:55:55.483-0700 [DEBUG] provider: using plugin: version=6
2024-01-12T16:55:55.483-0700 [DEBUG] provider.terraform-provider-pingone_v0.25.0: plugin address: address=/var/folders/lk/rdfj6whd5cd2vk0pkhv6zf9wfb1nt4/T/plugin738596251 network=unix timestamp=2024-01-12T16:55:55.483-0700
2024-01-12T16:55:55.507-0700 [DEBUG] provider.stdio: received EOF, stopping recv loop: err="rpc error: code = Unavailable desc = error reading from server: EOF"
2024-01-12T16:55:55.508-0700 [DEBUG] provider: plugin process exited: path=.terraform/providers/registry.terraform.io/pingidentity/pingone/0.25.0/darwin_arm64/terraform-provider-pingone_v0.25.0 pid=56389
2024-01-12T16:55:55.508-0700 [DEBUG] provider: plugin exited
2024-01-12T16:55:55.508-0700 [DEBUG] Building and walking validate graph
2024-01-12T16:55:55.508-0700 [DEBUG] ProviderTransformer: "pingone_notification_policy.Default_Notification_Policy" (*terraform.NodeValidatableResource) needs provider["registry.terraform.io/pingidentity/pingone"]
2024-01-12T16:55:55.508-0700 [DEBUG] ProviderTransformer: "pingone_group.identity_data_read_only" (*terraform.NodeValidatableResource) needs provider["registry.terraform.io/pingidentity/pingone"]
2024-01-12T16:55:55.508-0700 [DEBUG] ProviderTransformer: "data.pingone_role.identity_data_read_only" (*terraform.NodeValidatableResource) needs provider["registry.terraform.io/pingidentity/pingone"]
2024-01-12T16:55:55.508-0700 [DEBUG] ProviderTransformer: "data.pingone_population.ciam" (*terraform.NodeValidatableResource) needs provider["registry.terraform.io/pingidentity/pingone"]
2024-01-12T16:55:55.508-0700 [DEBUG] ProviderTransformer: "pingone_group.Identity_data_admin" (*terraform.NodeValidatableResource) needs provider["registry.terraform.io/pingidentity/pingone"]
2024-01-12T16:55:55.508-0700 [DEBUG] ProviderTransformer: "data.pingone_role.identity_data_admin" (*terraform.NodeValidatableResource) needs provider["registry.terraform.io/pingidentity/pingone"]
2024-01-12T16:55:55.508-0700 [DEBUG] ProviderTransformer: "pingone_group_role_assignment.population_identity_data_admin_to_group" (*terraform.NodeValidatableResource) needs provider["registry.terraform.io/pingidentity/pingone"]
2024-01-12T16:55:55.508-0700 [DEBUG] ProviderTransformer: "pingone_group_role_assignment.population_identity_data_ReadOnly_to_group" (*terraform.NodeValidatableResource) needs provider["registry.terraform.io/pingidentity/pingone"]
2024-01-12T16:55:55.508-0700 [DEBUG] pruning unused provider["registry.terraform.io/hashicorp/random"]
2024-01-12T16:55:55.508-0700 [DEBUG] ReferenceTransformer: "data.pingone_population.ciam" references: [var.pingone_environment_id]
2024-01-12T16:55:55.508-0700 [DEBUG] ReferenceTransformer: "output.identity_data_read_only_id (expand)" references: [pingone_group.identity_data_read_only]
2024-01-12T16:55:55.508-0700 [DEBUG] ReferenceTransformer: "pingone_notification_policy.Default_Notification_Policy" references: [var.pingone_environment_id]
2024-01-12T16:55:55.508-0700 [DEBUG] ReferenceTransformer: "pingone_group.identity_data_read_only" references: [var.pingone_environment_id]
2024-01-12T16:55:55.508-0700 [DEBUG] ReferenceTransformer: "data.pingone_role.identity_data_read_only" references: []
2024-01-12T16:55:55.508-0700 [DEBUG] ReferenceTransformer: "pingone_group.Identity_data_admin" references: [var.pingone_environment_id]
2024-01-12T16:55:55.508-0700 [DEBUG] ReferenceTransformer: "data.pingone_role.identity_data_admin" references: []
2024-01-12T16:55:55.508-0700 [DEBUG] ReferenceTransformer: "provider[\"registry.terraform.io/pingidentity/pingone\"]" references: []
2024-01-12T16:55:55.508-0700 [DEBUG] ReferenceTransformer: "pingone_group_role_assignment.population_identity_data_admin_to_group" references: [data.pingone_population.ciam var.pingone_environment_id pingone_group.Identity_data_admin data.pingone_role.identity_data_admin]
2024-01-12T16:55:55.508-0700 [DEBUG] ReferenceTransformer: "pingone_group_role_assignment.population_identity_data_ReadOnly_to_group" references: [pingone_group.identity_data_read_only data.pingone_role.identity_data_read_only data.pingone_population.ciam var.pingone_environment_id]
2024-01-12T16:55:55.508-0700 [DEBUG] ReferenceTransformer: "output.identity_data_admin_id (expand)" references: [pingone_group.Identity_data_admin]
2024-01-12T16:55:55.508-0700 [DEBUG] ReferenceTransformer: "var.pingone_environment_id" references: []
2024-01-12T16:55:55.509-0700 [DEBUG] Starting graph walk: walkValidate
2024-01-12T16:55:55.509-0700 [DEBUG] created provider logger: level=debug
2024-01-12T16:55:55.509-0700 [INFO]  provider: configuring client automatic mTLS
2024-01-12T16:55:55.512-0700 [DEBUG] provider: starting plugin: path=.terraform/providers/registry.terraform.io/pingidentity/pingone/0.25.0/darwin_arm64/terraform-provider-pingone_v0.25.0 args=[".terraform/providers/registry.terraform.io/pingidentity/pingone/0.25.0/darwin_arm64/terraform-provider-pingone_v0.25.0"]
2024-01-12T16:55:55.514-0700 [DEBUG] provider: plugin started: path=.terraform/providers/registry.terraform.io/pingidentity/pingone/0.25.0/darwin_arm64/terraform-provider-pingone_v0.25.0 pid=56390
2024-01-12T16:55:55.514-0700 [DEBUG] provider: waiting for RPC address: path=.terraform/providers/registry.terraform.io/pingidentity/pingone/0.25.0/darwin_arm64/terraform-provider-pingone_v0.25.0
2024-01-12T16:55:55.587-0700 [INFO]  provider.terraform-provider-pingone_v0.25.0: configuring server automatic mTLS: timestamp=2024-01-12T16:55:55.587-0700
2024-01-12T16:55:55.593-0700 [DEBUG] provider: using plugin: version=6
2024-01-12T16:55:55.593-0700 [DEBUG] provider.terraform-provider-pingone_v0.25.0: plugin address: address=/var/folders/lk/rdfj6whd5cd2vk0pkhv6zf9wfb1nt4/T/plugin3016510814 network=unix timestamp=2024-01-12T16:55:55.593-0700
2024-01-12T16:55:55.600-0700 [DEBUG] skipping FixUpBlockAttrs
2024-01-12T16:55:55.600-0700 [DEBUG] skipping FixUpBlockAttrs
2024-01-12T16:55:55.600-0700 [DEBUG] skipping FixUpBlockAttrs
2024-01-12T16:55:55.603-0700 [ERROR] provider.terraform-provider-pingone_v0.25.0: Response contains error diagnostic: diagnostic_severity=ERROR diagnostic_detail="Attribute quota list must contain at most 1 elements, got: 2" diagnostic_summary="Invalid Attribute Value" tf_proto_version=6.4 tf_resource_type=pingone_notification_policy tf_rpc=ValidateResourceConfig @caller=github.com/hashicorp/terraform-plugin-go@v0.20.0/tfprotov6/internal/diag/diagnostics.go:62 diagnostic_attribute="AttributeName(\"quota\")" tf_req_id=9c122e73-e298-5ed3-4cbd-fcffb640fd92 @module=sdk.proto tf_provider_addr=registry.terraform.io/pingidentity/pingone timestamp=2024-01-12T16:55:55.603-0700
2024-01-12T16:55:55.604-0700 [ERROR] vertex "pingone_notification_policy.Default_Notification_Policy" error: Invalid Attribute Value
╷
│ Error: Invalid Attribute Value
│ 
│   with pingone_notification_policy.Default_Notification_Policy,
│   on user_experience.tf line 1, in resource "pingone_notification_policy" "Default_Notification_Policy":
│    1: resource "pingone_notification_policy" "Default_Notification_Policy" {
│ 
│ Attribute quota list must contain at most 1 elements, got: 2
╵
2024-01-12T16:55:55.605-0700 [DEBUG] provider.stdio: received EOF, stopping recv loop: err="rpc error: code = Unavailable desc = error reading from server: EOF"
2024-01-12T16:55:55.606-0700 [DEBUG] provider: plugin process exited: path=.terraform/providers/registry.terraform.io/pingidentity/pingone/0.25.0/darwin_arm64/terraform-provider-pingone_v0.25.0 pid=56390
2024-01-12T16:55:55.606-0700 [DEBUG] provider: plugin exited

### Expected Behavior
Need to add SMS/Voice and Email quota limit

### Actual Behavior
Currently SMS/Voice quota limit only allowed not email 

### Steps to Reproduce

1. terraform plan

**Comment by patrickcping** [comment]
Hi @kasirajans !  Thanks for raising.  I agree that email quota limits should be added to this resource.  I've changed the type to an enhancement as the resource was originally created before email support was introduced on the API endpoint.

We'll target resolution for the next minor release

**Comment by kasirajans** [comment]
@patrickcping  thanks for the update

**Comment by patrickcping** [comment]
@kasirajans this was released today in version `v0.26.0`
